ytfzf

```diff
@@ -724,9 +724,11 @@ print_data () {
 get_video_format () {
 	# select format if flag given
 	if [ $show_format -eq 1 ]; then 
-		YTFZF_PREF="$(youtube-dl -F "$(printf "$selected_urls" | sed 1q)" | sed '1,3d' | sed '1!G; h; $!d' |\
-		eval "$menu_command" | sed -E 's/^([^ ]*) .*/\1/')"
-		[ -z "$YTFZF_PREF"  ] && exit;
+		        max_quality=$(youtube-dl -F "$(printf "$selected_urls")" | awk '{print $3}' | cut -dx -f2 | sort -n | tail -n1)
+        		quality=$(printf "Audio 144p 240p 360p 480p 720P 1080p 1440P 2160p" | awk -F"$max_quality""p" '{print $1 FS}' | sed "s/ /\n/g" | eval "$menu_command" | sed "s/p//g")
+			[ $quality = "Audio"  ] && YTFZF_PREF="bestaudio/best" || YTFZF_PREF="bestvideo[height=?$quality][fps<=?30][vcodec!=?vp9]+bestaudio/best"
+			[ -z "$YTFZF_PREF"  ] && exit;
```

Comment:
When there is no input in the format selection screen, suppose the user `ESC` on the selection screen, then we get a format error. A counter to this is adding the following line just after `quality` is declared
```
[ -z "$quality" ]  && exit
```
And we don't need line 730 after adding this.
Also make every variable other than `YTFZF_PREF` local.

Reply:
Thanks for the review. It's done.